Bottom: 938d1e267235384d46cea3a9c5957cf0a3331f2f
Top:    0c0ed37df04e3b3982c1c6dbd99241005b174938
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-04 17:26:52 -0500

Create new play for docker-hosts tasks

The majority of the docker container tasks belong to daedalus, while the
common docker tasks are for all docker hosts.


---
diff --git a/ansible-playbook/site.yml b/ansible-playbook/site.yml
index 5203a0e9..db9517d2 100644
--- a/ansible-playbook/site.yml
+++ b/ansible-playbook/site.yml
@@ -28,31 +28,26 @@
   - role: bacula-script
     assemble_scripts: false
   # - role: bacula-catalog
-- name: Install/Setup Docker images and containers
+
+- name: Prepare Docker host systems
   hosts: docker-hosts
   roles:
   - name: Setup SSH Daemon
     role: sshd
     tags:
     - sshd
-  - name: Run bacula-restores symlinks
-    role: bacula-restores
-    tags:
-    - bacula_restores
   - name: Run common Docker tasks
     role: docker-common
     tags:
     - docker_common
-    services:
-      - svn
-      - openldap
-      - exim4
-      - gitlab
-      - wiki
-      - mantisbt
-      - build
-      - testssh
-      - buildbot
+
+- name: Install/Setup Docker images and containers on daedalus
+  hosts: daedalus
+  roles:
+  - name: Run bacula-restores symlinks
+    role: bacula-restores
+    tags:
+    - bacula_restores
   - name: Configure Docker networking
     role: docker-networking
     net_connection: '{{ daedalus_net_connection }}'
